curriculum/challenges/english/25-front-end-development/lab-movie-review-page/67ef8f695db52583424bd118.md

```diff
@@ -0,0 +1,197 @@
+
+# --hints--
+
+You should have a `main` element.
+
+```js
+assert.isNotNull(document.querySelector("main"));
+```
+
+Inside the `main` element, you should have an `h1` element containing the movie title.
+
+```js
+const h1 = document.querySelector("main > h1");
+assert.isAbove(h1?.innerText.length, 0);
+```
+
+Below the `h1` element, you should have an `img` element with a descriptive `alt` attribute.
+  
+```js
+const img = document.querySelector("main > h1 + img");
+assert.isNotNull(img);
+assert.isNotEmpty(img.getAttribute("alt"));
+```
+
+You should have a `p` element after the image that contains the movie description.
+
+```js
+const description = document.querySelector("main img + p");
+assert.isNotNull(description);
+assert.isNotEmpty(description.innerText);
+```
+
+You should have another `p` element that contains the rating.
+
+```js
+const paragraphs = document.querySelectorAll("main p");
+assert.isAtLeast(paragraphs.length, 2);
+```
+
+Inside the second `p` element, you should have a `strong` element with the text `Movie Rating`.
+
+```js
+const strongEl = document.querySelector("main p:nth-of-type(2) strong");
+assert.match(strongEl?.innerText.trim(), /^Movie Rating:?$/);
+```
+
+There should be a `span` element inside the rating paragraph.
+
+```js
+const spanEl = document.querySelector("main p:nth-of-type(2) span");
+assert.isNotNull(spanEl);
+```
+
+The `span` element inside the rating paragraph should have ten stars, either filled in (⭐) or empty (☆).  
+
+```js  
+const spanEl = document.querySelector("main p:nth-of-type(2) span");  
+assert.match(spanEl?.innerText, /^(?:[⭐☆]{10})\s*\(\s*\d+\.\d+\s*\/\s*10\s*\)$/);
+```
+
+The `span` element inside the rating paragraph should have an `aria-hidden` attribute set to `true`.
+
+```js
+const spanEl = document.querySelector("main p:nth-of-type(2) span");
+assert.strictEqual(spanEl?.getAttribute("aria-hidden"), "true");
+```
+
+You should have an `h2` element with the text `Cast Members`. Your `h2` should come after your `h1`.
+
+```js
+const main = document.querySelector("main");
+const h1 = main?.querySelector("h1");
+const h2 = main?.querySelector("h2");
+
+assert.ok(h1);
+assert.ok(h2);
+assert.strictEqual(h2.innerText.trim(), "Cast Members");
+
+const elements = Array.from(main.querySelectorAll("*"));
+const h1Index = elements.indexOf(h1);
+const h2Index = elements.indexOf(h2);
+
+assert.isAbove(h2Index, h1Index);
+```
+
+You should have a `ul` element after the `h2`.
+
+```js
+const ul = document.querySelector("main h2 + ul");
+assert.isNotNull(ul);
+```
+
+The `ul` should contain multiple `li` elements.
+
+```js
+const liItems = document.querySelectorAll("main ul li");
+assert.isAbove(liItems.length, 1);
+```
+
+Each `li` should contain a `strong` element for the actor name and corresponding text for the character.
```

Comment:
since the tests expect campers to write the word `"as"` after the character that needs to be mentioned here just like in the instructions